internal/data/receiver_verification.go

```diff
@@ -75,6 +75,31 @@ func (m ReceiverVerificationModel) GetByReceiverIdsAndVerificationField(ctx cont
 	return receiverVerifications, nil
 }
 
+// GetAllByReceiverId returns all receiver verifications by receiver id.
+func (m ReceiverVerificationModel) GetAllByReceiverId(ctx context.Context, sqlExec db.SQLExecuter, receiverId string) ([]*ReceiverVerification, error) {
+	receiverVerifications := []*ReceiverVerification{}
```

Comment:
Is there any reason for returning an array of pointers []*ReceiverVerification{} rather than an array of values []ReceiverVerification{}?

I find it safer to deal with values rather than pointers. Especially in an array/slice where these values tend to be manipulated every now and then.